e2e-tests/cypress/support/submit-and-assert-radio-errors.js

```diff
@@ -5,9 +5,17 @@ import partials from '../e2e/partials';
  * @param {String} field
  * @param {Number} errorIndex - index of error in errorSummary
  * @param {Number} errorSummaryLength - the number of expected errors in errorSummary
+ * @param {Number} inlineError - the index of radios to find error
  * @param {String} errorMessage
  */
-export default (field, errorIndex, errorSummaryLength, errorMessage) => {
+export default (field, errorIndex, errorSummaryLength, errorMessage, inlineError) => {
+  let inlineErrorIndex = inlineError;
+
+  // if multiple radios on the page, need to know index to check inline error message
+  if (!inlineErrorIndex) {
+    inlineErrorIndex = 0;
```

Comment:
Could we have this in the function params as a default? Save having these lines of code